src/index.ts

```diff
@@ -1011,12 +1023,20 @@ export class ValidationService {
 
     private untrackFormInput(form: HTMLFormElement, inputUID: string) {
         let formUID = this.getElementUID(form);
-        if (!this.formInputs[formUID]) {
+        let formInputs = this.formInputs[formUID]
```

Comment:
I think this is a little confusing (as evidenced by the bug I point out later). As I understand it, `this.formInputs` is a way of looking up all form inputs for a form. And `formInputs` is all the inputs for the current form. So we may want to rename this local variable something like `currentFormInputs`.

And for extra credit, perhaps `this.formInputs` should be `this.formsInputs` or `this.forms`?

Reply:
For now I've applied consistency in usage/naming (`formInputUIDs = this.formInputs[formUID]`), which (combined with your "use the variable" suggestion) makes the bug more obvious.

A more precise rename (`inputsByFormUID`?) might be useful, but there are several similar lookup fields we'd want to update for consistency.